### PR TITLE
bump @opengovsg/starter-kitty-validators to 1.2.10

### PR DESCRIPTION
**Pull request by adriangohjw**
## Problem

re: https://opengovproducts.slack.com/archives/CK68JNFHR/p1730972525340969

## Solution

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
  - Details ...
- [ ] No - this PR is backwards compatible

- pump dependencies in package.json and package-lock.json
- polyfill unsupported `node:path` for webpack in nextjs and storybook

**New dependencies**:

- `@opengovsg/starter-kitty-validators` : bumped to 1.2.7

### CODE REVIEW

File: apps/studio/.storybook/main.ts [hunk 1]
```diff
@@ -25,6 +26,16 @@ const config: StorybookConfig = {
     disableTelemetry: true,
   },
 
+  // https://github.com/vercel/next.js/issues/28774#issuecomment-1264555395
```
Comment (harishv7) [inline apps/studio/.storybook/main.ts:29]: are these changes related to the version bump?

File: apps/studio/next.config.mjs [hunk 2]
```diff
@@ -65,6 +67,15 @@ const config = {
   images: {
     domains: [env.NEXT_PUBLIC_S3_ASSETS_DOMAIN_NAME ?? ""].filter((d) => d),
   },
+  // https://github.com/vercel/next.js/issues/28774#issuecomment-1264555395
```
Comment (harishv7) [inline apps/studio/next.config.mjs:70]: saw the linked issue, why did this occur suddenly?

**Comment by adriangohjw** [comment]
@harishv7 [v1.2.3](https://github.com/opengovsg/starter-kitty/releases/tag/v1.2.3) introduced breaking changes. Checked against v1.1.3 and it was still fine

**Review by harishv7 — APPROVED** [review]
Pls check if any breaking changes before merging!

**Comment by adriangohjw** [comment]
@harishv7 just an update - the above webpack issue is fixed in https://github.com/opengovsg/starter-kitty/pull/32 (v 1.2.9) so i've proceeded to remove the polyfill
